### DISCUSSION TITLE
What does Irradiance do?

### DISCUSSION THREAD

**Discussion by WenhuaJ** [o3de/o3de#17345] [Q&A]
**I have a question. Both BasePBR and StandardPBR materials have the Irradiace parameter, but I didn't notice a difference when I modified it. Does it work now? Is it useless? Why does it exist?**
**I hope someone can answer my doubts.**

**Comment by ohmaya** [comment]
https://development--o3deorg.netlify.app/docs/atom-guide/look-dev/materials/pbr/
Describes how a surface interacts with the global illumination (GI) system’s diffuse lighting environment. It does not affect the appearance of the material itself.
